Cargo.lock

```diff
@@ -119,7 +119,7 @@ name = "bigint"
 version = "3.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "byteorder 1.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "byteorder 1.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
do you need 1.1 version update for this pr?
better keep version updates separately just in case to revert easy